static/index.html

```diff
@@ -6,6 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <title>ECL Twig</title>
+    <script>
+      if (window.location.hash.match('access_token')) {
```

Comment:
You can surely have the checks in an or statement or `[...list].includes(match)` way of combining the several checks

Reply:
Yep, we can look for ways to simplify this.